library/axi_dac_interpolate/axi_dac_interpolate_constr.xdc

```diff
@@ -0,0 +1,4 @@
+set_false_path  -to [get_cells -hier -filter {name =~ *trigger_i_m1_reg* && IS_SEQUENTIAL}]
+set_false_path  -to [get_cells -hier -filter {name =~ *trigger_adc_m1_reg* && IS_SEQUENTIAL}]
+set_false_path  -to [get_cells -hier -filter {name =~ *trigger_la_m1_reg* && IS_SEQUENTIAL}]
```

Comment:
Synchronization registers should have an active ASYNC_REG attribute.

Reply:
Added. Thanks!